README.md

```diff
@@ -15,27 +16,27 @@ This demo created for quick-testing original models for mpii dataset (other data
 1. Download required models from original repository from step 8 of Installation
```

Comment:
Add link on one example (to not go to another repo), e.g. [386x386 model](path-to-model).